langchain4j-scoring/src/main/test/java/dev/langchain4j/model/scoring/onnx/OnnxScoringModelIT.java

```diff
@@ -0,0 +1,72 @@
+package dev.langchain4j.model.scoring.onnx;
+
+import ai.onnxruntime.OrtSession;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.scoring.ScoringModel;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.IOException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.data.Percentage.withPercentage;
+
+class OnnxScoringModelIT {
+
+    @TempDir
+    private static Path tempDir;
+
+    private static ScoringModel model;
+
+    @BeforeAll
+    static void initModel() throws IOException {
+
+        // I set up a local proxy to download the model
+        // System.setProperty("https.proxyHost","127.0.0.1" );
+        // System.setProperty("https.proxyPort","7890" );
+
+        URL modelUrl = new URL("https://huggingface.co/Xenova/ms-marco-MiniLM-L-6-v2/resolve/main/onnx/model.onnx?download=true");
```

Comment:
nit: what about using [model_quantized.onnx](https://huggingface.co/Xenova/ms-marco-MiniLM-L-6-v2/blob/main/onnx/model_quantized.onnx) instead? It is even smaller == faster build

Reply:
ok